src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -352,6 +352,7 @@ function BaseSelectionList<TItem extends ListItem>(
                 onCheckboxPress={onCheckboxPress ? () => onCheckboxPress?.(item) : undefined}
                 onDismissError={() => onDismissError?.(item)}
                 shouldPreventDefaultFocusOnSelectRow={shouldPreventDefaultFocusOnSelectRow}
+                shouldPreventEnterKeySubmit
```

Comment:
I find it a bit confusing becuase you can still select the row with enter, but the props suggests it should be prevented

Reply:
Could we add a comment to the prop where we pass it as `true` to explain this? Thanks @allroundexperts

Reply:
Done